README.md

```diff
@@ -91,6 +91,7 @@ Trouble comes with the following defaults:
     },
     multiline = true, -- render multi-line messages
     indent_lines = true, -- add an indent guide below the fold icons
+    win_border = "single", -- border style for floating windows. Accepts same border values as |nvim_open_win()|.
```

Comment:
can you make this win_config? So we can allow a user to set any other things as well if they'd like